internal/services/postgres/validate/flexible_server_storage_tier.go

```diff
@@ -4,6 +4,9 @@
 package validate
```

Comment:
I assume that we don't need it anymore if using "IsExplicitlyNullInConfig" to fix the issue.